vite.config.ts

```diff
@@ -21,13 +21,7 @@ readdirSync(documentModelsDir, { withFileTypes: true })
     });
 
 export default defineConfig(({ mode }) => {
-    const external = [
-        'document-model',
```

Comment:
If we do this then we should move them from `peerDependencies` to `dependencies`.

Reply:
yeah good idea 👍 will do

Reply:
Actually, devEependencies.
Since they are going to be included in the bundle they don't need to be installed